webapp/sources/rudder/rudder-core/src/main/scala/com/normation/cfclerk/domain/TechniqueVersion.scala

```diff
@@ -78,27 +77,27 @@ object TechniqueVersion {
    * NOTE: it seems that in existing env, sometimes epoch was written when it should
    * not have. So we must accept also leading number+":".
    */
-  protected def apply(v: Version): Either[String, TechniqueVersion] = {
```

Comment:
I'm not all against default parameters, but do you think we may forget some revision some times ? (but i can imagine how many lines it saved you to edit)

Reply:
It was necessary to avoid breaking most of the code. I can imagine removing the default value in first patch release or next minor